### PR TITLE
Fix roster push when new contact is pushed from server and emit rosterPush event

### PR DESCRIPTION
**Pull request by teseo**
Fix for the onRosterPush method and emitting a rosterPush event in to allow developers custom their applications with the new rosterPush

### CODE REVIEW

File: converse.js [hunk 1]
```diff
@@ -4375,7 +4375,10 @@
                 $(iq).children('query').find('item').each(function (idx, item) {
                     this.updateContact(item);
                 }.bind(this));
-                return true;
+
+                converse.emit('rosterPush', iq);
+
```
Comment (jcbrand) [inline converse.js]: Please document this emitted event, here: https://github.com/jcbrand/converse.js/blob/master/docs/source/development.rst#event-types

File: converse.js [hunk 2]
```diff
@@ -4362,7 +4362,7 @@
                  */
                 var id = iq.getAttribute('id');
                 var from = iq.getAttribute('from');
-                if (from && from !== "" && Strophe.getNodeFromJid(from) != converse.bare_jid) {
+                if (from && from !== "" && from != converse.connection.jid) {
                     // Receiving client MUST ignore stanza unless it has no from or from = user's bare JID.
```
Comment (jcbrand) [inline converse.js]: Please use "!==" instead of "!=".

**Comment by teseo** [comment]
Thank you for the advise Jc. There you have the requested changes. Hope all is good.

Pleasure collaborating on converse.js

**Comment by jcbrand** [comment]
Thanks @teseo!

**Pull request merged**